flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSourceITCase.java

```diff
@@ -112,7 +117,8 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void testStopWithSavepoint() throws Exception {
+    void testStopWithSavepoint(@InjectClusterClient RestClusterClient<?> clusterClient)
```

Comment:
```suggestion
    void testStopWithSavepoint(@InjectClusterClient RestClusterClient<?> clusterClient, @TempDir Path tmp)
```
nit: We could also just path the temporary directory into this method instead of having it as a field variable. ...since it's the only location where the temporary directory is used. But that's just personal taste, I guess. :thinking: